docs/pages/access-controls/idps.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Configure Teleport as an identity provider
+description: How to set up Teleport's identity provider functionality
+---
+
+Since Teleport 12.1 users can authenticate to both internal and external applications
+through the use of a built in identity provider.
```

Comment:
Should we mention that it's in preview mode on this page as well?

Reply:
Ahhh yes, good move. Will do!